test-suite-java-spring/src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+server:
+  port: 8701
+  servlet:
+    encoding:
+      force-response: true
```

Comment:
is `server.servlet.encoding.force-response` required for micronaut open api?

Reply:
no, of course not. Removed it